apps/wallet-mobile/src/features/Send/common/SendContext.tsx

```diff
@@ -185,7 +185,7 @@ const targetsReducer = (state: SendState, action: TargetAction) => {
           if (index === selectedTargetIndex) {
             if (addressRecords !== undefined) {
               const keys = Object.keys(addressRecords).filter(isNameServer)
-              const nameServer = keys.length === 1 ? keys[0] : undefined
+              const nameServer = keys[0]
```

Comment:
AFAIK this was to automatically select IFF there was only 1 record found, otherwise, if there were 2, or more, it will ASK the user to select, was this behaviour changed by product? we will select automatically if more than 1 record is found?

Reply:
hi @stackchain 

Finally product has decided to go ahead with this change until a final decision is made